etna/generic_pages/models.py

```diff
@@ -3,13 +3,13 @@
 
 from etna.core.models import BasePage
 
-from .blocks import AboutPageStreamBlock
+from .blocks import GeneralPageStreamBlock
 
 
-class AboutPage(BasePage):
-    """About Page"""
+class GeneralPage(BasePage):
+    """GeneralPage"""
```

Comment:
This docstring is not useful in it's current state, so it'd probably be better to remove it

Reply:
Done